.github/workflows/release.yml

```diff
@@ -11,6 +11,9 @@ permissions:
   actions: read
   checks: write
 
+env:
+  IMAGE: mcr.microsoft.com/azurelinux/base/core:3.0
```

Comment:
It would be nice to find a way to share this between the release and ci pipelines, but I have not found a way to do that so far.